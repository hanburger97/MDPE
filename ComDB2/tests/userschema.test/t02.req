set user mohit
set password mohit
put password 'abcd' for 'abcd'
put password 'dcba' for 'dcba'
select * from comdb2_users
