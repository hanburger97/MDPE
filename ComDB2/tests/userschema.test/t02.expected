(username='abcd', isOP='N')
(username='dcba', isOP='N')
(username='default', isOP='N')
(username='mohit', isOP='Y')
